system/ee/legacy/libraries/Filemanager.php

```diff
@@ -308,12 +308,17 @@ function get_image_dimensions($file_path)
 		{
 			$D = @getimagesize($file_path);
 
-			$image_size = array(
-				'height'	=> $D['1'],
-				'width'	=> $D['0']
+			if($D && is_array($D)) {
+
+				$image_size = array(
```

Comment:
Can switch to short array syntax

`$image_size = [ ... ]`

Reply:
Updated in latest commits.